code/datums/craft/recipes/armor.dm

```diff
@@ -320,4 +320,13 @@
 		list(CRAFT_MATERIAL, 4, MATERIAL_PLASTEEL), // In accordance to flak vests
 		list(QUALITY_WELDING, 15, 30),
 		list(QUALITY_ADHESIVE, 15, 30)
-	)
\ No newline at end of file
+	)
+/datum/craft_recipe/armor/breach2
```

Comment:
It seems like it's always a no brainer to upgrade given how easy it is to make this.